app/views/create-client-build.html

```diff
@@ -236,6 +236,18 @@ <h3 class="with-divider">Source Configuration</h3>
                 <div class="section">
                   <h3 class="with-divider">Build Configuration</h3>
                   <dl class="dl-horizontal left">
+
+                    <div class="form-group">
+                      <label for="build-type">Platform</label>
+                      <ui-select required input-id="build-platform" ng-model="newClientBuild.buildPlatform" search-enabled="false">
+                        <ui-select-match>{{buildTypeMap[$select.selected].label}}</ui-select-match>
+                        <ui-select-choices
+                            repeat="platform in buildTypeMap[newClientBuild.clientType].buildPlatforms">
+                          {{buildTypeMap[platform].label}}
+                        </ui-select-choices>
+                      </ui-select>
+                    </div>
```

Comment:
This is currently being displayed all the time, even for iOS and Android client types which will only have the 1 platform. Did it this way to make the form consistent, but we can easily only display it for cordova or for client types that support more than one platform if preferred.

Reply:
can we only show this field if `buildPlatforms` has more than one item?

Reply:
I would prefer just showing this to clients that support more than one platform. I don't think it would add any value to the form otherwise.

Reply:
I have no opinion on this either way tbh, but if others do, we can hide it based on the length of the buildPlatforms array.